### PR TITLE
Create com.github.alecadd.sequeler

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build com.github.alecadd.sequeler

**Comment by bilelmoussaoui** [comment]
How does the buildbot doesn't detect the appdata file when it's there? 
https://github.com/Alecaddd/sequeler/blob/master/data/com.github.alecaddd.sequeler.appdata.xml And with the correct file name

**Comment by Alexander-Wilms** [comment]
It needs to be installed to /share/appdata/. Lookin at the CMakeLists.txt, it looks like it's not being installed. This is how it should be done: https://github.com/clintbellanger/flare-game/commit/d6608c4f5d3cecbb9bd6d9d616cafb571b27d49d

File: com.github.alecadd.sequeler.json [hunk 1]
```diff
@@ -0,0 +1,184 @@
+{
+  "app-id": "com.github.alecadd.sequeler",
```
Comment (Alexander-Wilms) [inline com.github.alecadd.sequeler.json]: The third "d" is missing

Comment (bilelmoussaoui) [reply com.github.alecadd.sequeler.json]: Fixed

**Comment by bilelmoussaoui** [comment]
@nedrichards Can you trigger another build, please? Just fixed the app-id & the filename. The rest of the modules builds correctly. The app works pretty fine on GNOME without the elementary theme. Tried it on Fedora Rawhide

**Comment by bilelmoussaoui** [comment]
Builds successfully https://flathub.org/builds/#/builders/2/builds/309 :100:

**Comment by bilelmoussaoui** [comment]
@nedrichards Anything else that should be done here?

**Comment by nedrichards** [comment]
@bilelmoussaoui similar to other apps from the Elementary ecosystem, does that need the elemetary theme and icons to be installed and set by default? Other than that, lgtm

**Comment by bilelmoussaoui** [comment]
It's on Fedora 28 default repos and works correctly. I will try again and let you know this evenings

**Comment by bilelmoussaoui** [comment]
screenshot from 2018-03-27 11-24-26

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.alecadd.sequeler

**Pull request closed without merging**